<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio Controls Editor</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AnimatedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="m_pCentralWidgetLayout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>972</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSave_All"/>
    <addaction name="actionReload"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QDockWidget" name="m_pInspectorDockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="m_pInspectorWidget">
    <layout class="QVBoxLayout" name="m_pInspectorDockLayout">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="m_pMiddlewareDockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Audio Middleware Controls</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="m_pMiddlewareWidget">
    <layout class="QVBoxLayout" name="m_pMiddlewareDockLayout">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
    </layout>
   </widget>
  </widget>
  <action name="actionSave_All">
   <property name="icon">
    <iconset resource="AudioControlsEditorUI.qrc">
     <normaloff>:/AudioControlsEditor/Icons/Save_Icon.png</normaloff>:/AudioControlsEditor/Icons/Save_Icon.png</iconset>
   </property>
   <property name="text">
    <string>Save All</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="AudioControlsEditorUI.qrc">
     <normaloff>:/AudioControlsEditor/Icons/Load_Icon.png</normaloff>:/AudioControlsEditor/Icons/Load_Icon.png</iconset>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="AudioControlsEditorUI.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionSave_All</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>Save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionReload</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>Reload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>CurrentControlNameChanged(QString)</signal>
  <signal>CurrentControlEditingFinished()</signal>
  <slot>Reload()</slot>
  <slot>SelectedInternalControlChanged(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
  <slot>UpdateAddConnectionMenu(int)</slot>
  <slot>SelectedConnectionChanged(QListWidgetItem*,QListWidgetItem*)</slot>
  <slot>Save()</slot>
  <slot>NewLibrary()</slot>
  <slot>CreateSwitchControl()</slot>
  <slot>CreateTriggerControl()</slot>
  <slot>CreateRTPCControl()</slot>
  <slot>AddEnvironmentsLibrary()</slot>
  <slot>CreateEnvironmentsControl()</slot>
  <slot>UpdateSelectedConnection()</slot>
  <slot>AddPreloadLibrary()</slot>
  <slot>CreatePreloadControl()</slot>
  <slot>CurrentConnectionGroupChanged(QString)</slot>
  <slot>SetScopeForCurrentControl(QString)</slot>
  <slot>SetSelectedControlName(QString)</slot>
  <slot>SetAutoLoadForCurrentControl(bool)</slot>
  <slot>UpdateControlData()</slot>
  <slot>ShowExtraColumns(bool)</slot>
 </slots>
</ui>
